From 8c130b7a5d83148b552ef600ee1cc795e4823dd2 Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Thu, 13 Aug 2026 13:31:27 -0400
Subject: [PATCH 1/3] don't build grpcio in install_all_python_modules.sh

We run this script in CIBW_TEST_COMMAND to install grpcio's other Python
packages and their test dependencies, but we don't want it to reinstall
grpcio itself from source - we want the test environment to use the
wheel built (and about to be distributed) in this same cibuildwheel
invocation.

This is a rework of the older patches from Mark to handle upstream's
change to using pyproject.toml instead of setup.py.

Upstream-Status: Inappropriate [upstream tests in a different way]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 tools/distrib/install_all_python_modules.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/distrib/install_all_python_modules.sh b/tools/distrib/install_all_python_modules.sh
index 5652580c76..acab4f9d6f 100755
--- a/tools/distrib/install_all_python_modules.sh
+++ b/tools/distrib/install_all_python_modules.sh
@@ -30,7 +30,7 @@ function maybe_run_command () {
 }
 
 python3 -m pip install --upgrade "cython==3.1.1";
-python3 -m pip install .;
+#python3 -m pip install .;
 
 # Build and install grpcio_tools
 pushd tools/distrib/python/grpcio_tools;
-- 
2.55.0

